src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-otel.mdx

```diff
@@ -17,11 +17,11 @@ OpenTelemetry observability for Kubernetes provides complete, open-source setup
 
 This document outlines the process for monitoring a Kubernetes cluster using OpenTelemetry. It involves the installation of the [`nr-k8s-otel-collector`](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector) Helm chart within the cluster and the deployment of the necessary collectors to enable first-class observability.
```

Comment:
`the deployment of the necessary Collectors to enable first-class observability.`